### ISSUE TITLE
Telegram notifications from AWX

### ISSUE THREAD

**Issue by amg-web** [ansible/awx#8317]
- Feature Idea
Please add Telegram Notifications to AWX

Now available in AWX 15.0:
email, Grafana, IRC, Mettermost, Pagerduty, Rocket.chat, Slack, Twilio, webhook

**Comment by Colonizator1** [comment]
You can easily use the webhook type.
Example:
- target url: `https://api.telegram.org/botYOUR_BOT_ID/sendMessage`
- HTTP Headers: `{ "Content-Type": "application/json" }`
- HTTP Method: POST
- Customize messages… **turn on**
- Success message body (or start/error): 
```
{
    "chat_id": "YOUR_CHAT_ID",
    "text": "success - \"{{ job_friendly_name }}\""
}
```

**Comment by souovan** [comment]
> You can easily use the webhook type. Example:
> 
>     * target url: `https://api.telegram.org/botYOUR_BOT_ID/sendMessage`
> 
>     * HTTP Headers: `{ "Content-Type": "application/json" }`
> 
>     * HTTP Method: POST
> 
>     * Customize messages… **turn on**
> 
>     * Success message body (or start/error):
> 
> 
> ```
> {
>     "chat_id": "YOUR_CHAT_ID",
>     "text": "success - \"{{ job_friendly_name }}\""
> }
> ```

Hi @Colonizator1 does this still work ? i've tried without success

**Comment by Colonizator1** [comment]
@souovan Hi, we are using such telegram notifications right now. Any errors? Did you check your request manually, with postman, for example?

**Comment by souovan** [comment]
Is there a way to debbug this on AWX ? i only got this when i test the notification 
![image](https://user-images.githubusercontent.com/42985246/232486632-907ea13a-e16e-45ae-8e6a-008435a732f0.png)

**Comment by Colonizator1** [comment]
If you are sure that your request URL and data are correct, try to attache notification to some simple template.
I guess you tried to test the notification with the Test button. I failed with it too. But it works well for templates.

**Comment by souovan** [comment]
Yeah, i've tried with the Test button, i will try attaching on some template.

**Comment by LeoSpyke** [comment]
Any news on this?

**Comment by Colonizator1** [comment]
@LeoSpyke it's working as webhook type, read my message above

**Comment by amg-web** [comment]
it works strange
I can not edit text message, it does not save it.

**Comment by LeoSpyke** [comment]
> @LeoSpyke it's working as webhook type, read my message above

Yes I've seen your message, but I mean that this is not an integration, rather it is a "collateral" workaround that works.
Another native approach would allow to write the token in a specific password field, like the Token for Twilio or Pagerduty, rather than in plain text in the Target URL field.

**Comment by igorferrati** [comment]
Maybe this issue can help you:

[webhook notification body is not valid anymore](https://github.com/ansible/awx/issues/14985)
